mod-infra/src/main/scala/at/forsyte/apalache/infra/passes/options.scala

```diff
@@ -585,6 +603,34 @@ object OptionGroup extends LazyLogging {
     }
   }
 
+  /** Options used to configure trace evaluation. */
+  case class Tracee(
+      trace: SourceOption,
+      expressions: List[String])
+      extends OptionGroup
+
+  object Tracee extends Configurable[Config.Tracee, Tracee] with LazyLogging {
+    def apply(tracee: Config.Tracee): Try[Tracee] = {
+
+      def validateExprs(lst: List[String]): Try[List[String]] =
+        if (lst.nonEmpty) Success(lst)
+        else Failure(new PassOptionException("Trace evaluation requires a nonempty list of expressions."))
+
+      def validateSource(sourceOption: SourceOption): Try[SourceOption] = sourceOption match {
+        case FileSource(_, SourceOption.Format.Itf | SourceOption.Format.Json) => Success(sourceOption)
+        case _ => Failure(new PassOptionException("Trace evaluation requires an ITF or JSON trace."))
```

Comment:
I think we've also discussed this elsewhere, but why not also support string inputs?

```suggestion
      def validateSource(sourceOption: SourceOption): Try[SourceOption] = sourceOption.format match {
        case SourceOption.Format.Itf | SourceOption.Format.Json => Success(sourceOption)
        case _ => Failure(new PassOptionException("Trace evaluation requires an ITF or JSON trace."))
```

Reply:
String inputs don't tell you if the insides should follow ITF or ApalacheJSON formatting.

Reply:
I think they do, unless i misunderstand. See https://github.com/informalsystems/apalache/blob/e03bc42258aa844869794bf5efc7bb440628db93/mod-infra/src/main/scala/at/forsyte/apalache/infra/passes/options.scala#L302

It has exactly the same format info as `FileSource`.